File: Demo/src/main/java/org/alee/demo/skin/NightOption.java
package org.alee.demo.skin;

import android.os.Build;

import org.alee.component.skin.service.IThemeSkinOption;

import java.util.LinkedHashSet;

/**********************************************************
 *
 * @author: MY.Liu
 * @date: 2021/2/19
 * @description: xxxx
 *
 *********************************************************/
final class NightOption implements IThemeSkinOption {
    @Override
    public LinkedHashSet<String> getStandardSkinPackPath() {
        LinkedHashSet<String> set = new LinkedHashSet<>();
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
            set.add("/storage/emulated/0/Android/data/org.alee.demo.skin/files/Documents/night.skin");
        } else {
            set.add("/sdcard/night.skin");
        }
        return set;
    }
}

File: BasicDemo/src/main/java/org/alee/demo/skin/basic/ability/NightOption.kt
package org.alee.demo.skin.basic.ability

import org.alee.component.skin.service.IThemeSkinOption
import org.alee.demo.skin.basic.ability.util.loadBoolean

/**
 * 夜色模式配置
 *
 * <p> 详细描述
 *
 * @author MingYu.Liu
 * created in 2022/9/13
 *
 */
internal class NightOption : IThemeSkinOption {

    override fun getStandardSkinPackPath(): LinkedHashSet<String> {
        return LinkedHashSet<String>().apply {
            // 基础黑夜皮肤包
            add(SKIN_PACK_PATH + NIGHT_SKIN_PACK_NAME)
        }.apply {
            // 春节皮肤包
            if (KEY_USE_SPRING_FESTIVAL_SKIN.loadBoolean(false)) {
                add(SKIN_PACK_PATH + SPRING_FESTIVAL_SKIN_PACK_NAME)
            }
        }
    }
}